backend/app/Http/Controllers/FollowController.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\User;
+use App\Models\FollowLog;
+use Illuminate\Http\Request;
+use Illuminate\Validation\Rule;
+use Illuminate\Support\Facades\DB;
+
+class FollowController extends Controller {
+    public function index(Request $request) {
+        $id = $request->user()->id;
+        return $this->getUserWithFollows($id);
```

Comment:
There is a facade in [laravel for authenticated users](https://laravel.com/docs/7.x/authentication#retrieving-the-authenticated-user), `Auth` or `auth()`

```suggestion
        return $this->getUserWithFollows(Auth::id());
```